NBitcoin/Protocol/Node.cs

```diff
@@ -522,52 +539,68 @@ public static Node Connect(Network network, NodeConnectionParameters parameters
 		/// </summary>
 		/// <param name="network">The network to connect to</param>
 		/// <param name="parameters">The parameters used by the found node, use AddressManagerBehavior.GetAddrman for finding peers</param>
-		/// <param name="connectedEndpoints">Function returning the already connected endpoints, the new endpoint will be select outside of existing groups</param>
+		/// <param name="connectedEndPoints">Function returning the already connected endpoints, the new endpoint will be select outside of existing groups</param>

```

Comment:
I believe endpoint is one word (no need for c case)

Reply:
Fixed